rest/model/project.go

```diff
@@ -27,7 +27,7 @@ type APIProject struct {
 func (apiProject *APIProject) BuildFromService(p interface{}) error {
 	v, ok := p.(model.ProjectRef)
 	if !ok {
-		return fmt.Errorf("incorrect type when fetching converting project type")
+		return fmt.Errorf(fmt.Sprintf("incorrect type when fetching converting project type %T", p))
```

Comment:
errorf has the same signature as sprintf, so you don't need to use sprintf here.